src/redux/constants.js

```diff
@@ -53,3 +53,5 @@ export const fontBoundary = {
   max: 56,
   min: 8,
 };
+
+export const sourceForCodeMirror = 'source';
```

Comment:
I'm not a huge fan of this name. Doesn't really explain what the constant is for. Perhaps something like `LOCAL_STORAGE_SOURCE_KEY`?

Maybe add a comment explaining what it is for?